NAMESPACE

```diff
@@ -1,315 +1,322 @@
-import(stats)
-importFrom(lattice,
-           xyplot,
-           qqmath)
-importFrom(car,
-           powerTransform,
-           bcPower)
-importFrom(Hmisc,
-           wtd.quantile)
-import(zoo)
-importFrom(graphics,plot)
-import(latticeExtra)
-import(polynom)
-import(reshape)
-
-## Exports from imported packages
-export(simulate,fitted,residuals,predict,update,coef)
-export(xyplot)
-# export(summary,print)
-export(plot)
-
-## CORE FUNCTIONS
-
-export(hydromad)
-S3method(print, hydromad)
-S3method(fitted, hydromad)
-S3method(residuals, hydromad)
-S3method(observed, hydromad)
-S3method(vcov, hydromad)
-S3method(logLik, hydromad)
-S3method(deviance, hydromad)
-S3method(str, hydromad.runlist)
-export(isValidModel)
-S3method(isValidModel, default)
-S3method(isValidModel, hydromad)
-S3method(isValidModel, runlist)
-S3method(isValidModel, tf)
-
-S3method(predict, hydromad)
-
-S3method(update, hydromad)
-
-S3method(coef, hydromad)
-
-export(absorbScale)
-S3method(absorbScale, hydromad)
-
-S3method(simulate, hydromad)
-
-export(parameterSets)
-export(rotatedSampling)
-
-S3method(summary, hydromad)
-S3method(summary, hydromad.runlist)
-S3method(print, summary.hydromad)
-S3method(print, summaryWithBreaks.hydromad)
-
-S3method(plot, hydromad)
-S3method(xyplot, hydromad)
-S3method(xyplot, hydromad.runlist)
-S3method(qqmath, hydromad)
-S3method(tsdiag, hydromad)
-
-
-## ROUTING MODELS
-
-export(tryModelOrders)
-S3method(summary, tryModelOrders)
-
-export(makePrefilter, ## remove this?
-       estimateDelay,
-       estimateDelayFrac,lagFrac
-       )
-
-export(tfParsConvert)
+# Generated by roxygen2: do not edit by hand
 
-export(armax.sim,
-       ssg.armax,
-       normalise.armax,
-       armax.ls.fit,
-       armax.sriv.fit,
-       armax.inverse.fit,
-       armax.inverse.sim)
-useDynLib(hydromad, sriv_system)
-useDynLib(hydromad, inverse_filter)
-
-export(expuh.sim,
-       ssg.expuh,
-       normalise.expuh,
-       expuh.ls.fit,
-       expuh.sriv.fit,
-       expuh.inverse.fit,
-       expuh.inverse.sim,
-       maexpuh.sim
-)
-
-export(filter_loss) # export?
-useDynLib(hydromad, filter_constloss)
-
-export(expuh3s.sim,leakyExpStore.sim)
-
-export(powuh.sim,
-       ssg.powuh,
-       normalise.powuh)
-
-export(lambda.sim,
-       ssg.lambda,
-       normalise.lambda,
-       lambda.inverse.sim,
-       lambda.inverse.fit)
-useDynLib(hydromad, inverse_filter_lambda)
-
-export(deconvolution.uh)
-
-## tf.R
-S3method(print, tf)
-S3method(coef, tf)
-S3method(update, tf)
-S3method(predict, tf)
-S3method(fitted, tf)
-S3method(observed, tf)
-S3method(xyplot, tf)
-S3method(vcov, tf)
-
-
-## SOIL MOISTURE ACCOUNTING MODELS
-
-export(scalar.sim)
-S3method(absorbScale,hydromad.scalar)
-
-export(cwi.sim)
+S3method("[",runlist)
+S3method(absorbScale,hydromad)
 S3method(absorbScale,hydromad.cwi)
```

Comment:
This might be an unrealistic request, but would there be an easy way of giving a run down of what additions and deletions have been made to the NAMESPACE?
Given the tests are mostly passing, I assume it's mostly ok, but for a big PR like this one, it would add confidence to know what has changed.
Perhaps a simple way would be to sort the original NAMESPACE file in the same order as roxygen does it?

Reply:
I constructed a VLOOKUP in Excel that tells you which exports from the new NAMESPACE match the original. I'm happy to email you the file, will do so now.